src/main/java/gov/cabinetofice/gapuserservice/service/OneLoginService.java

```diff
@@ -167,8 +168,8 @@ public StateCookieDto decodeStateCookie(final String stateCookie) {
     public String generateAndStoreState(final HttpServletResponse response, final String redirectUrl, final String saltId) {
         final String state = this.generateState();
         final String encodedStateJsonString = this.buildEncodedStateJson(redirectUrl, state, saltId);
-        String STATE_COOKIE = "state";
-        final Cookie stateCookie = WebUtil.buildSecureCookie(STATE_COOKIE, encodedStateJsonString, 3600);
+        String stateCookieValue = "state";
```

Comment:
This is the cookie name, not the value. We should probably define this as a static member of the class as we're doing with other constant values:

```
private static final String STATE_COOKIE = "state";
```